src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/Application.js

```diff
@@ -31,6 +32,14 @@ export default class Application extends React.Component<Props> {
         this.toggleNavigation();
     };
 
+    handleLogoutClick = () => {
+        // TODO: Logout user here.
+    };
+
+    handleProfileEditClick = () => {
+        // TODO: Open profile edit overlay here.
+    };
```

Comment:
These two functions are not used anymore, looks like you've made a wrong choice during the rebase